pkgs/by-name/en/endlines/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+  stdenv.mkDerivation (finalAttrs: {
+    pname = "endlines";
+    version = "1.9.2";
+
+    src = fetchFromGitHub {
+      owner = "mdolidon";
+      repo = "endlines";
+      rev = finalAttrs.version;
+      hash = "sha256-M0IyY/WXR8qv9/qx5G0pG3EKqMoZAP3fJTZ6sSSMMyQ=";
+    };
+
+    prePatch = ''
+      sed -i "s@/usr/local@$out@g" Makefile
+    '';
+
+    preInstall = ''
+      mkdir -p $out/bin
+    '';
+
+    meta = with lib; {
+      homepage = "https://github.com/mdolidon/endlines";
+      description = "Easy conversion between new-line conventions";
+      license = licenses.asl20;
+      maintainers = with maintainers; [zedseven];
+      mainProgram = "endlines";
+      platforms = platforms.unix;
```

```suggestion
    meta = {
      homepage = "https://github.com/mdolidon/endlines";
      description = "Easy conversion between new-line conventions";
      license = lib.licenses.asl20;
      maintainers = with lib.maintainers; [ zedseven ];
      mainProgram = "endlines";
      platforms = lib.platforms.unix;
```

https://nix.dev/guides/best-practices.html#with-scopes